From de06f50eee35e3ea0daaf77d21e8888f8d8753ad Mon Sep 17 00:00:00 2001
From: Paul Crawford <psc@sat.dundee.ac.uk>
Date: Wed, 21 Dec 2016 22:00:51 +0000
Subject: [PATCH 03/10] Ignore build products in GIT

- Added a .gitignore file to list those files we do not track in GIT

- Based on pull request by Maciej S. Szmigiero:

  Currently, after doing checkout and build 'git status' on the
  source tree returns a lot of untracked files. Their presence
  makes it harder to spot actual tree changes, so let's make
  the noise disappear from 'git status' by adding proper .gitignore
  files.
---
 .gitignore | 38 ++++++++++++++++++++++++++++++++++++++
 1 file changed, 38 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..db2baeb
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,38 @@
+# Ignore hidden/backup/compilation files
+
+.*
+*.o
+*~
+
+# However don't ignore this file
+
+!/.gitignore
+
+# Ignore the build programs
+
+src/watchdog
+src/wd_identify
+src/wd_keepalive
+
+# Ignore remaining files create/modified by the steps:
+# autoreconf -i
+# ./configure 
+# make
+
+Makefile
+Makefile.in
+aclocal.m4
+autom4te.cache/
+compile
+config.log
+config.status
+configure
+depcomp
+include/config.h
+include/config.h.in
+include/stamp-h1
+install-sh
+missing
+src/Makefile
+src/Makefile.in
+
-- 
2.20.1

